### PR TITLE
adding README

### PR DESCRIPTION
**Pull request by michelengelen**
adds a basic README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    ```
+
+2. import `ThemeProvider` in your application root
+
+    In order for the compass components to be syled correctly you need to wrap your applcation in a `ThemeProvider`.
+
+    It also manages custom themes for your application (if you wish so).
+
+    ```javascript
+    import ThemeProvider from '@mattermost/compass-components/utilities/theme';
```
Comment (nevyangelova) [inline README.md:24]: optional: we can show an example of how to wrap their application as well?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+3. Run storybook
+
+    once all packages are installed you can run storybook:
+
+    ```bash
+    npm run storybook
+    ```
+
+    when storybook is up and running you can access it by navigating in your browser to `localhost:6006`.
```
Comment (nevyangelova) [inline README.md:69]: optional: maybe its a good idea to describe the component generator too ?

**Review by nevyangelova — COMMENTED** [review]
Thank you for adding this 🎉   Two optional comments/questions :)

File: README.md (hunk 2 above)
Comment (michelengelen) [reply README.md:69]: Absolutely

File: README.md (hunk 1 above)
Comment (michelengelen) [reply README.md:24]: Will add this on Monday 👍🏻

**Review by nevyangelova — APPROVED** [review]
Noisee 🎉

**Pull request merged**